vignettes/saving_serializing.Rmd

```diff
@@ -0,0 +1,341 @@
+---
+title: "Saving and serializing models"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Saving and serializing models} 
+  %\VignetteEngine{knitr::rmarkdown} 
+  %\VignetteEncoding{UTF-8}
+type: docs
+repo: https://github.com/rstudio/keras
+menu:
+  main:
+    name: "Saving and serializing models"
+    identifier: "saving-serializing"
+    parent: "keras-advanced"
+    weight: 70
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+> This tutorial is an R translation of [this page](https://www.tensorflow.org/alpha/guide/keras/saving_and_serializing)
+available in the official TensorFlow documentation.
+
+The first part of this guide covers saving and serialization for Sequential models and 
+models built using the Functional API and for Sequential models. The saving and serialization 
+APIs are the exact same for both of these types of models.
+
+Saving for custom subclasses of Model is covered in the section "Saving Subclassed Models". 
+The APIs in this case are slightly different than for Sequential or Functional models.
+
+## Setup
+
+```{r}
+library(keras)
+```
+
+## Saving Sequential Models or Functional models
+
+```{r}
+inputs <- layer_input(shape = 784, name = "digits")
+outputs <- inputs %>% 
+  layer_dense(units = 64, activation = "relu", name = "dense_1") %>% 
+  layer_dense(units = 64, activation = "relu", name = "dense_2") %>% 
+  layer_dense(units = 10, activation = "softmax", name = "predictions")
+model <- keras_model(inputs, outputs) # can't name the model
```

Comment:
can't name means, from our side? If so, should we make this more explicit and/or investigate? Or do we just need to add a parameter?

Reply:
Yes, it's from our side. I'll open another PR for this